services/web/server/src/simcore_service_webserver/users_exceptions.py

```diff
@@ -0,0 +1,54 @@
+"""Defines the different exceptions that may arise in the projects subpackage"""
+
+
+class UsersException(Exception):
+    """Basic exception for errors raised in projects"""
+
+    def __init__(self, msg=None):
+        if msg is None:
+            msg = "Unexpected error occured in projects subpackage"
+        super(UsersException, self).__init__(msg)

```

Comment:
Python3 is less verbose regarding ``super``. According to https://docs.python.org/3/library/functions.html#super, you can write this code as
```python
def __init__(self, msg=None):
      super().__init__(msg or "Unexpected error .." )
```

same applies to all code below